examples/rect/README.md

```diff
@@ -11,11 +11,9 @@ theme: {
     '24': ['24px', '24px'],
```

Comment:
while you're in here, this line is too indented

Reply:
Oops will fix

Reply:
Something must be up w/ Github previews. This is what I see:
![image](https://user-images.githubusercontent.com/283397/165540137-b86416e5-4530-4248-9e23-7b27b5c551c9.png)